### ISSUE TITLE
OverflowError in Windows

### ISSUE THREAD

**Issue by dasolyn** [fizyr/keras-retinanet#64]
Line 89 in /keras_retinanet/preprocessing/pascal_voc.py,

```python
seed = np.uint32(time.time() * 1000)
```

This line always raise OverflowError in Windows but linux don't.

**Comment by erhwenkuo** [comment]
According to [What is bit size of lon on 64-bit Window?](https://stackoverflow.com/questions/384502/what-is-the-bit-size-of-long-on-64-bit-windows) , apparently the issue is caused by the data type definition between Windows and *nix-like system.

After checking the logic, I think the purpose of below code block is to get a random seed which would keep `keras.preprocessing.image.Iterator` to have reproduciable iteration patten.

Code before change:
`
seed = np.uint32(time.time() * 1000)
`

I think it should be ok to remodify the code block like below:
`
seed = np.uint32(time.time())
`

**Comment by hgaiser** [comment]
Resolved in https://github.com/fizyr/keras-retinanet/pull/48.
